Google Maps API/credentials.js

```diff
@@ -0,0 +1,4 @@
+window.hereCreds = {
```

Comment:
You should not post your api keys to github, usually you put them in an .env file which does not upload to github, then call when needed